docs/assisting-with-translations.md

```diff
@@ -68,3 +65,90 @@ translations at once:
     
     # Push changes back to Transifex
     tx push -s -t -f --skip --no-interactive
+    
+___
+
+## Getting started with the Translation Team
+
+While the website itself is managed on [GitHub](https://github.com/bitcoin-dot-org/bitcoin.org/), the translations are handled through [Transifex](https://www.transifex.com/bitcoinorg/bitcoinorg/).
+This section shall explain the first steps on Transifex for new volunteers who want to help with translating bitcoin.org.
+
+General guidelines provided by Transifex itself can be found [here](https://docs.transifex.com/getting-started/translators).
+
+### 1. Create a Transifex Account
+* Follow this [link](https://www.transifex.com/signup/) and create an account. Not much information is needed.
+
+### 2. Login and join the Bitcoin.org Translation Team
+
+* Follow this [link](https://www.transifex.com/bitcoinorg/bitcoinorg/) and click on “Join team”.
+* Select the language you want to translate into.
+* Your request to join a team will be accepted instantly, and you will be a translator for the language you selected.
+* If your language is not available yet, close the pop-up, scroll down, and click on “Request language” on the right side. Please consider first if it is necessary that bitcoin.org is available in your language. There is a lot of content to be translated and especially for languages with a low number of native speakers, there won’t be many people available that are willing to assist you.
+
+### 3.	Get to know the Interface
```

Comment:
Please fix spacing.

Reply:
I can not observe this issue. Also the preview file looks OK. Can you please check?

Reply:
Alright, could observe it a few minutes ago. Removed space and added it again. Should be fine now.